test/jasmine/tests/hover_pie_test.js

```diff
@@ -186,44 +185,60 @@ describe('pie hovering', function() {
 
         afterEach(destroyGraphDiv);
 
-        it('should show the default selected values', function(done) {
-
-            var expected = ['4', '5', '33.3%'];
-
-            Plotly.plot(gd, mockCopy.data, mockCopy.layout).then(function() {
+        function _hover() {
+            mouseEvent('mouseover', 223, 143);
+        }
 
-                mouseEvent('mouseover', 223, 143);
+        function assertLabel(expected) {
+            var labels = d3.selectAll('.hovertext .nums .line');
 
-                var labels = Plotly.d3.selectAll('.hovertext .nums .line');
+            expect(labels[0].length).toBe(expected.length);
```

Comment:
done in https://github.com/plotly/plotly.js/pull/1523/commits/1bf54e29a966dd8a203c9bd0f9dd022f3928183e